### ISSUE TITLE
psal (MediaWiki 1.32) reports pages downloaded but not edit

### ISSUE THREAD

**Issue by nemobis** [WikiTeam/wikiteam#353]
```
Titles saved at... psalcsdrexeledu-20200208-titles.txt
1376 page titles loaded
https://psal.cs.drexel.edu/api.php
Retrieving the XML for every page from "start"
Downloaded 10 pages
Downloaded 20 pages
Downloaded 30 pages
Downloaded 40 pages
Downloaded 50 pages
Downloaded 60 pages
Downloaded 70 pages
Downloaded 80 pages
Downloaded 90 pages
Downloaded 100 pages
Downloaded 110 pages
Downloaded 120 pages
```

Will there be any revisions actually? Special:Export online works fine.

**Comment by nemobis** [comment]
Fixed in 9ec6ce42d33d2dcaa12d59e0389467e839e30ace : etree needed some more checks before trying to add non-existing fields.

**Comment by nemobis** [comment]
Actually that was unrelated, it's still doing this when not using `--xmlrevisions`. On the other hand the XML is downloading fine.
